### PR TITLE
OPENEUROPA-1746

### PR DESCRIPTION
**Pull request by voidtek**
### Description

- [x]  Change the format of variable names on configuration files.
- [x]  Fix issue on hash_salt on the settings during the setting install.

### CODE REVIEW

File: config/commands/clone.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ command:
...
       options:
-        asda-url: ${toolkit.clone.asda-url}
-        asda-user: ${env.ASDA_USER}
-        asda-password: ${env.ASDA_PASSWORD}
+        asda_url: ${toolkit.clone.asda_url}
+        asda_user: ${env.ASDA_USER}
+        asda_password: ${env.ASDA_PASSWORD}
         dumpfile: ${toolkit.clone.dumpfile}
-        project-id: ${toolkit.project-id}
+        project_id: ${toolkit.project_id}
```
Comment (ademarco) [inline config/commands/clone.yml]: Nope, so the options are with `-` while the configuration is with `_` :smile:

File: config/default.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 toolkit:
-  project-id: 'PROJECTID'
+  project_id: 'PROJECTID'
   clone:
-    asda-url: 'https://webgate.ec.europa.eu/fpfis/files-for/automate_dumps'
+    asda_url: 'https://webgate.ec.europa.eu/fpfis/files-for/automate_dumps'
```
Comment (ademarco) [inline config/default.yml:4]: These are ok.

File: src/TaskRunner/Commands/CloneCommands.php [hunk 3]
```diff
@@ -82,27 +82,27 @@ public function installDump(array $options = [
...
   public function downloadDump(array $options = [
-    'asda-url' => InputOption::VALUE_REQUIRED,
-    'asda-user' => InputOption::VALUE_REQUIRED,
-    'asda-password' => InputOption::VALUE_REQUIRED,
+    'asda_url' => InputOption::VALUE_REQUIRED,
+    'asda_user' => InputOption::VALUE_REQUIRED,
+    'asda_password' => InputOption::VALUE_REQUIRED,
     'dumpfile' => InputOption::VALUE_REQUIRED,
-    'project-id' => InputOption::VALUE_REQUIRED,
+    'project_id' => InputOption::VALUE_REQUIRED,
```
Comment (ademarco) [inline src/TaskRunner/Commands/CloneCommands.php]: Same as above, options should be with `-`

File: tests/Commands/DrupalCommandsTest.php [hunk 4]
```diff
@@ -2,71 +2,82 @@
...
+  }
+}
+
+namespace EcEuropa\Toolkit\TaskRunner\Commands {
+
+  /**
+   * Override random_bytes function for test.
+   */
+  function random_bytes() {
+    return 'abc';
```
Comment (ademarco) [inline tests/Commands/DrupalCommandsTest.php:80]: Shouldn't this be `return 'YWJj'`?

**Review by ademarco — CHANGES_REQUESTED** [review]

File: tests/Commands/DrupalCommandsTest.php (hunk 4 above)
Comment (voidtek) [reply tests/Commands/DrupalCommandsTest.php:80]: Because it's the result of `str_replace(['+', '/', '='], ['-', '_', ''], base64_encode('abc'))`.

Comment (ademarco) [reply tests/Commands/DrupalCommandsTest.php:80]: right! :D

**Review by ademarco — APPROVED** [review]

**Review by jonhy81 — APPROVED** [review]

**Pull request merged**